### PR TITLE
Contract release 1.0.0

### PR DESCRIPTION
**Pull request by fleupold**
Contains deployments for release!

Note that while deployment is not deterministic the contracts happen to have the same addresses for this release (this cannot be relied on).

All deployments were verified (mainnet etherscan & gnosisscan doesn't show it yet, I will monitor if this is just a caching issue)

- Barn (0xD02De8Da0B71E1B59489794F423FaBBa2AdC4d93)
  - [Goerli](https://goerli.etherscan.io/address/0xD02De8Da0B71E1B59489794F423FaBBa2AdC4d93#code)
  - [GChain](https://blockscout.com/xdai/mainnet/address/0xD02De8Da0B71E1B59489794F423FaBBa2AdC4d93/contracts#address-tabs)
  - [Mainnet](https://etherscan.io/address/0xd02de8da0b71e1b59489794f423fabba2adc4d93#code)
- Prod (0x40A50cf069e992AA4536211B23F286eF88752187)
  - [Goerli](https://goerli.etherscan.io/address/0x40A50cf069e992AA4536211B23F286eF88752187#code)
  - [GChain](https://blockscout.com/xdai/mainnet/address/0x40A50cf069e992AA4536211B23F286eF88752187/contracts#address-tabs)
  - [Mainnet](https://etherscan.io/address/0x40A50cf069e992AA4536211B23F286eF88752187#code)

### CODE REVIEW

**Comment by alfetopito** [comment]
Awesome!

Just one comment: Shouldn't updating `networks.[prod|barn].json` be part of this PR?

**Comment by fleupold** [comment]
> Just one comment: Shouldn't updating networks.[prod|barn].json be part of this PR?

I believe this will be auto-generated by CI once this commit is tagged as a release

File: .gitignore [hunk 1]
```diff
@@ -5,11 +5,11 @@ out/
 # Ignores development broadcast logs
 !/broadcast
 /broadcast/*/31337/
-/broadcast/Deploy.sol/5/*
-!/broadcast/Deploy.sol/5/deployment*.json
+/broadcast/Deploy.sol/*/*
```
Comment (anxolin) [inline .gitignore:8]: probably you don't need the double *, right?

**Review by anxolin — APPROVED** [review]
Thanks! 

ubernit: the name of the branch looks more like a tag than a branch. If you want to make it a release branch, maybe `release/1.0` is better. Then, once we tag the close release, we create the tag `v1.0.0`

**Comment by fedgiac** [comment]
> ubernit: the name of the branch looks more like a tag than a branch. If you want to make it a release branch, maybe release/1.0 is better. Then, once we tag the close release, we create the tag v1.0.0

Just wanted to mention that having a branch with the same name as a tag would not cause issue in our release script (we used the same name in the previous releases just to make sure it works, so for consistency this name makes sense).

**Comment by fleupold** [comment]
I closed this branch in favor of #45 (which uses a better branch name)

**Pull request closed without merging**